.templates/new-module/src/js/react.jsx

```diff
@@ -0,0 +1,18 @@
+/*! [replace-name] v[replace-version] */
```

Comment:
Yeah pancake stitches them all together but will retain the comments so you can know from looking at the code what models and versions you used.